modules/core/src/main/scala/org/tessellation/http/routes/DagRoutes.scala

```diff
@@ -38,6 +38,13 @@ final case class DagRoutes[F[_]: Async](dagService: DAGService[F], mkDagCell: L0
         case _ => NotFound()
       }
 
+    case GET -> Root / "total-wallets" =>
```

Comment:
Maybe "total-wallets-count" or just "wallets-count" for me "total-wallets" sounds confusing.

Reply:
I think the more english version is
`user-count`
(singular noun)